src/pool/srv.c

```diff
@@ -25,6 +25,10 @@ init(void)
 {
 	int rc;
 
+	bool use_swim = true;
+	d_getenv_bool("DAOS_USE_SWIM", &use_swim);
+	pool_disable_exclude = !use_swim;
```

Comment:
what is pool_disable_exclude variable for? i only see it being declared and set, but not used

Reply:
It controls whether PSs responds to SWIM info. See src/pool/srv_pool.c.

Reply:
thanks, i didnt notice at first that this variable was already there, just changed from static to extern